src/porepy/models/solution_strategy.py

```diff
@@ -300,6 +300,10 @@ def set_materials(self):
         fluid = cast(
             pp.FluidComponent, constants.get("fluid", pp.FluidComponent(name="fluid"))
         )
+        numerical = cast(
```

Comment:
Please adapt also the comment in line 295 regarding the numerical constants.

Reply:
Done